src/redactable.rs

```diff
@@ -0,0 +1,19 @@
+/// Types that are sensitive data or PII (Personally Identifiable Information) and can be redact-formatted.
+///
+/// The type must have an implementation of [`Display`](std::fmt::Display) so that it can also be formatted in plain text without redaction.
+///
+/// This trait can be derived using the [`Redactable`](derive.Redactable.html) macro.
+pub trait Redactable: std::fmt::Display {
```

Comment:
Does this trait need to have Display as a supertrait? Isn't that only a requirement for the derive macro?

Reply:
Hmmm good point - I suppose it is only a requirement for the derive macro. We can remove this supertrait.